docs/reference/coordinates/index.rst

```diff
@@ -85,247 +83,6 @@ and `Franz & Harper (2002) <https://doi.org/10.1016/S0032-0633(01)00119-2>`_
 (and `corrected version <https://www2.mps.mpg.de/homes/fraenz/systems/systems3art/systems3art.html>`_).
 
```

Comment:
I feel like we should link to the guide pages here?

Reply:
Fixed.